Android.mk

```diff
@@ -251,11 +251,11 @@ ifeq ($(ANDROID_MAJOR),$(filter $(ANDROID_MAJOR),8 9))
 LOCAL_SHARED_LIBRARIES += android.hardware.camera.provider@2.4
 endif
 
-ifeq ($(ANDROID_MAJOR),$(filter $(ANDROID_MAJOR),10))
+ifeq ($(ANDROID_MAJOR),$(filter $(ANDROID_MAJOR),10 12))
```

Comment:
Is this really needed and the change to the entry below is not enough?

Reply:
Seems to be redundant for both, removed.